Prox/Prox/RemoteConfigDefaults.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>significant_location_change_distance_meters</key>
+	<integer>100</integer>
```

Comment:
This will need to be rebased on James' changes where the default value is specified in code.